src/pages/studyView/StudyViewPage.tsx

```diff
@@ -163,17 +161,17 @@ export default class StudyViewPage extends React.Component<IStudyViewPageProps,
     get groupsButton() {
         return (
             <>
-                <If condition={!isWebdriver()}>
-                    <InfoBeacon
-                        top={-15}
-                        right={45}
-                        interaction={'mouseover'}
-                        color={'green'}
-                        id={'groupComparison1'}
-                    >
-                        <WrappedTour/>
-                    </InfoBeacon>
-                </If>
+                {/*<If condition={!isWebdriver()}>*/}
```

Comment:
leaving this in as poor man's documentation for our next usage of this.   where else does this belong?  ReadME? Wiki?

Reply:
Could maybe add it here in a new section? https://docs.cbioportal.org/4.-development/release-procedure. Prolly good to add some comment above this as well, like "uncomment to show a green info beacon at the top of the page, useful to highlight a new feature defined in Usermessage.tsx".